peachjam/models/judgment.py

```diff
@@ -72,10 +72,31 @@ def __str__(self):
         return self.name
 
 
+class CourtRegistry(models.Model):
+    court = models.ForeignKey(
+        Court,
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="court_registry",
```

Comment:
This should be plural, eg for `court.court_registries`

Reply:
Actually, I think `registries` would be better. We have `judgment.registry`, it should use similar naming.